### ISSUE TITLE
Archive this repo

### ISSUE THREAD

**Issue by basnijholt** [snoack/flake8-per-file-ignores#11]
To make it clearer that this package is no longer needed.

**Comment by v-goncharenko** [comment]
@basnijholt  Why is it so?

Is there a way to ignore wildcarded files in `flake8` natively?

**Comment by basnijholt** [comment]
Yes, see https://github.com/snoack/flake8-per-file-ignores/blame/master/README.md#L6-L8.

@snoack, the fact that this isn't obvious to @v-goncharenko is precisely the reason to archive this 😄

**Comment by v-goncharenko** [comment]
Or at least write it with BIGGER LETTERS and some alert signs ⚠️
